tests/test_wave_01.py

```diff
@@ -2,52 +2,65 @@
 # The following line imports the Vendor class from the module vendor inside the swap_meet package.
 from swap_meet.vendor import Vendor
 
-
-def test_vendor_has_inventory():
+# TEST 1 = PASSED
+# @pytest.mark.skip(reason="no way of currently testing this")
+def test_vendor_has_inventory(): # has
+    # arrange, act - 
```

Comment:
This isn't a big deal but in the future, it'd be preferred that changes to the tests don't get pushed. You can make sure they never get pushed by never using `git add` on them in the first place. In other words by specifying specific files instead of using `git add .`